code/datums/objectives/target/debrain.dm

```diff
@@ -0,0 +1,30 @@
+/datum/objective/target/debrain/find_target() //I want braaaainssss
+	..()
+	if(target && target.current)
+		explanation_text = "Steal the brain of [target.current.real_name]."
+	else
+		explanation_text = "Free Objective"
+	return target
+
+/datum/objective/target/debrain/find_target_by_role(role, role_type=FALSE)
+	..(role, role_type)
+	if(target && target.current)
+		explanation_text = "Steal the brain of [target.current.real_name] the [!role_type ? target.assigned_role : target.special_role]."
+	else
+		explanation_text = "Free Objective"
```

Comment:
/datum/objective's default explanation_text could be this, then you wouldn't have to put this else clause in every place with a dynamic explanation next